core/src/test/resources/com/google/googlejavaformat/java/testdata/I603.input

```diff
@@ -0,0 +1 @@
+sealed abstract class T permits D, E {}
```

Comment:
Maybe add a couple more test cases, like one that has both a `permits` clause and `extends` or `implements`, and one where the signature doesn't all fit on one line?

Reply:
Yes, that makes sense.  It just struck me now that these tests will fail if not run on a JDK 15 or later.  Should I handle this in `core/pom.xml` and exclude them if that’s the case?

Reply:
The logic in `FormatterIntegrationTest.java` will ensure these tests are only run on JDK >= 15. They aren't compiled by maven it just sees them as resources. I don't think an exclusion in the pom is necessary.

Reply:
Yes, that’s right, my mistake.